recipes/forest_lite/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "forest_lite" %}
+{% set version = "0.16.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MetOffice/forest-lite/archive/v{{ version }}.tar.gz
+  sha256: 24103013a84aad2303986186ce6d328dcabf3e659a81ed88919de9328f12ecfe
+
+build:
+  noarch: python
```

Comment:
Are the `build_js` platform specific? I don't know but if yes this cannot be noarch.

Reply:
@ocefpaf I'm not 100% familiar with building npm on different operating systems but since we are targetting the browser and not node I suspect this will be platform independent

Reply:
OK, then `noarch` is fine.